src/webportal/src/app/user/fabric/user-profile/storage-list.jsx

```diff
@@ -43,6 +43,15 @@ function getStorageServerUri(storageDetail) {
           {data.containerName}; <b>{'Path: '}</b>
         </>
       );
+    case 'dshuttle':
+      return (
+        <>
+          {'AzureBlob '}
```

Comment:
Can we show AzureBlob only when ufsType is `wasb`? For other type we can show the raw `ufsUri`

Reply:
Done.